src/server.ts

```diff
@@ -8,6 +8,9 @@ import { usersRouter } from './routes/user.route.js';
 import { productRouter } from './routes/productRoutes.js';
 import { sellerRouter } from './routes/sellerRoutes.js';
 import { rolesRouter } from './routes/roleRoutes.js';
+import cartRooutes from './routes/cartRoutes.js';
+import cartRoutes from './routes/cartRoutes.js';
```

Comment:
you have the same imports in the same file and there is no where you are using it in that file if you are not using it you may remove it

Reply:
corrected